src/nordjylland_news/summary_data_set.py

```diff
@@ -0,0 +1,135 @@
+import os
```

Comment:
Det plejer at være god stil at have en modul-docstreng i toppen af hvert modul, som kort beskriver hvad modulet indeholder. Det kommer også automatisk med i dokumentationen.
```suggestion
"""Class that builds and contains the summarisation dataset."""

import os
```